README.md

```diff
@@ -16,6 +16,16 @@ The keys in the `env` secret are exposed in the `checkout` and `command` as envi
 
 ### AppRole Authentication
 
+## Environment Hook
+```bash
+# This is configuration for setting the vault token, as an example to the environment
+echo "~~~ :buildkite: agent :vault: config"
+export VAULT_TOKEN="buildkite-is-cool"
+
+# Alternatively, you can use secret-env to store the VAULT_SECRET_ID, instead of the below line
```

Comment:
This is still confusing as is, it probably makes more sense to have two examples, one with just setting `VAULT_SECRET_ID` and one setting the secret-env, like:

```bash
# This value is set in your agent's Environment hook
SUPER_SECRET_ID=$(vault read -field "secret_id" auth/approle/role/buildkite/secret-id)
```
```yaml
steps:
  - command: ./run_build.sh
    plugins:
      - vault-secrets#v2.0.0:
          server: "https://my-vault-server"
          path: secret/buildkite
          auth:
            method: "approle"
            role-id: "my-role-id"
            secret-env: "SUPER_SECRET_ID"
```